Frontend/.env.example

```diff
@@ -1,2 +1,2 @@
-DEPLOYED_CHAIN_ID=1337
-ALCHEMY_ID=""
\ No newline at end of file
+REACT_APP_DEPLOYED_CHAIN_ID=1337
```

Comment:
For me it works by changing ID here to `4` 
Maybe they changed the chain ids for testnets suddenly in Ethereum? There are lots of changes nowadays

Reply:
Yeah, we've deployed to rinkeby testnet (4). I just used my local node (1337) for testing and forgot to mention it though there's an `.env.example` but will add comments on the README

Reply:
I think it is better to change the id on the `.env.example` to 4